src/main/resources/templates/components/project-frameset.ftl

```diff
@@ -1,10 +1,15 @@
-[#macro renderBreadcrumb i18n group]
-    <ol class="breadcrumb hidden-xs">
-        <li><a href="/courses">${ i18n.translate("section.courses") }</a></li>
-        <li><a href="${group.course.course.getURI()}">${group.course.course.code} - ${group.course.course.name}</a></li>
-        <li><a href="${group.course.getURI()}">${group.course.timeSpan.start?string["yyyy"]}[#if group.course.timeSpan.end??] - ${group.course.timeSpan.end?string["yyyy"]}[/#if]</a></li>
-        <li class="active">Group ${group.getGroupNumber()}</li>
-    </ol>
+[#macro renderBreadcrumb i18n group repository]
+	<ol class="breadcrumb hidden-xs">
+		[#if group?? && group?has_content]
+			<li><a href="/courses">${ i18n.translate("section.courses") }</a></li>
+			<li><a href="${group.course.course.getURI()}">${group.course.course.code} - ${group.course.course.name}</a></li>
+			<li><a href="${group.course.getURI()}">${group.course.timeSpan.start?string["yyyy"]}[#if group.course.timeSpan.end??] - ${group.course.timeSpan.end?string["yyyy"]}[/#if]</a></li>
+			<li class="active">Group ${group.getGroupNumber()}</li>
+		[#elseif repository?? && repository?has_content]
```

Comment:
We call a method on `repositoryEntity`, but check if `repository` is present. This needs some normalization 😄  Because this sanity check can give false positives.